src/js/components/DataTableColumns/__tests__/__snapshots__/DataTableColumns-test.tsx.snap

```diff
@@ -877,8 +881,8 @@ exports[`DataTableColumns remove column 1`] = `
   border-bottom: solid 1px rgba(0, 0, 0, 0.12);
   padding-left: 12px;
   padding-right: 12px;
-  padding-top: 6px;
-  padding-bottom: 6px;
+  padding-top: 12px;
```

Comment:
Changes only due to jsdom, does not appear in browser. Correct value is re-assigned below